src/CLI/TargetPhpVersion.php

```diff
@@ -17,10 +17,10 @@ class TargetPhpVersion
         '8.1',
         '8.2',
         '8.3',
+        '8.4',
```

Comment:
iirc here we are defining which version arkitect is capable of parsing. Should we keep versions older than 8.0? if yes, should then keep testing them (at least smoke test the phar) to be sure we are actually able to do so?